src/common.ts

```diff
@@ -167,4 +167,25 @@ export function deleteInstallFile(type: InstallFileType): Promise<void> {
 export function convertNativePathToPosix(pathString: string): string {
     let parts = pathString.split(path.sep);
     return parts.join(path.posix.sep);
+}
+
+/**
+ * This function checks to see if a subfolder is part of folder.
+ * 
+ * Assumes subfolder and folder are absolute paths.
+ * 
+ * @param subfolder subfolder to check if it is part of the folder parameter
+ * @param folder folder to check aganist
+ */
+export function isSubfolderOf(subfolder: string, folder: string): boolean {
+    if (os.platform() !== 'win32')
```

Comment:
Did you mean to do something here?

Reply:
Assuming your idea was to try and solve the case sensitivity problem, here is my thought - I don't think we should try to solve it. From what I can tell, I think the paths we are dealing with are all from the VS Code search APIs using the same roots they are providing to us, so it seems like it would be very surprising if we saw the same path but with different case. Solving the casing problem in java script is likely a hard problem. For one because on both OSX and Linux its possible to have either case sensitivity or insensitivity (though OSX is generally insensitive and Linux is sensitive). For another because I don't know how closely the java script casing tables are actually going to match what the OS uses.

My suggestion: Just add a note to the header that the two paths must have consistent casing.

Reply:
> My suggestion: Just add a note to the header that the two paths must have consistent casing.

yup. That's what I was thinking too.